### ISSUE TITLE
How to execute the sh script ?

### ISSUE THREAD

**Issue by Vponed** [nix-community/nix-user-chroot#98]
Three commands:
/home/Program/nix-user-chroot /myuser/.nix  konsole  
home/myuser/.nix-profile/bin/nix-shell -p tor 
tor -f torrc

What I haven't done, but I can't get them to run inside sh

**Comment by Mic92** [comment]
Can you rephrase your question? I do not quite understand what you want to achieve.

**Comment by Vponed** [comment]
> Не могли бы вы перефразировать свой вопрос? Я не совсем понимаю, чего вы хотите добиться.

I want these three commands to be executed in sh-scratch, without requiring manual input. so far they can only be executed sequentially
or so:
_/home/Program/nix-user-chroot   /myuser/.nix  nix-shell -p tor --command "tor -f torrc"_
sh scripts can't be made to work

**Comment by Mic92** [comment]
Does this work for you?

```
 /home/Program/nix-user-chroot /myuser/.nix bash -c 'nix-shell -p tor --run "tor -f torrc"'
```

**Comment by Vponed** [comment]
> Работает ли это для вас?
> 
> ```
>  /home/Program/nix-user-chroot /myuser/.nix bash -c 'nix-shell -p tor --run "tor -f torrc"'
> ```

yes!

**Comment by Mic92** [comment]
Ok. Problem solved. The program you pass to `nix-user-chroot` needs to exist outside the sandbox, which is not the case for `nix-shell` but for bash.
